docs/getting-started/continuous_integration.md

```diff
@@ -94,6 +95,67 @@ make sure to set the dry-run parameters in both the `Build Fuzzers` and `Run Fuz
 limit for broken fuzz targets than OSS-Fuzz's check_build. Most users should
 not set this.
 
+#### Adding Other Sanitizers
+CIFuzz supports address, memory and undefined sanitizers. Address is the default
+sanitizer and will be used for every job in which a sanitizer is not specified.
+To add another sanitizer to your workflow copy the `Fuzzing` job and rename it
+to the sanitizer you want to fuzz with. Then add the sanitizer variable to both
+the `Build Fuzzers` step and the `Run Fuzzers` step. The choices are `'address'`,
+`'memory'`, and `'undefined'`. Once this additional job is configured the CIFuzz
+workflow will run all of the jobs corresponding to each sanitizer simultaneously.
+It is important to note that the `Build Fuzzers` and the `Run Fuzzers` sanitizer
+field needs to be the same. See the following main.yml file for an example.
+
+```yaml
+name: CIFuzz
+on: [pull_request]
+jobs:
+ AddressFuzzing:
+   runs-on: ubuntu-latest
+   steps:
+   - name: Build Fuzzers
+     uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
+     with:
+       oss-fuzz-project-name: 'example'
+       dry-run: false
+       # sanitizer: address
```

Comment:
Is this commented out because ASAN is the default?

Reply:
I'm not sure. That part was copied verbatim from https://github.com/google/oss-fuzz/pull/3516 by reverting the revert and resolving a bunch of conflicts. To be honest I'd replace this example with something like
```yaml
jobs:
 Fuzzing:
   runs-on: ubuntu-latest
   strategy:
     fail-fast: false
     matrix:
       sanitizer: [address, undefined, memory]
   steps:
   - name: Build Fuzzers (${{ matrix.sanitizer }})
     id: build
     uses: evverx/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
     with:
       oss-fuzz-project-name: 'systemd'
       dry-run: false
       allowed-broken-targets-percentage: 50
       sanitizer: ${{ matrix.sanitizer }}
   - name: Run Fuzzers (${{ matrix.sanitizer }})
     uses: evverx/oss-fuzz/infra/cifuzz/actions/run_fuzzers@master
     with:
       oss-fuzz-project-name: 'systemd'
       fuzz-seconds: 600
       dry-run: false
       sanitizer: ${{ matrix.sanitizer }}
   - name: Upload Crash
     uses: actions/upload-artifact@v1
     if: failure() && steps.build.outcome == 'success'
     with:
       name: ${{ matrix.sanitizer }}-artifacts
       path: ./out/artifacts
```
With the matrix, I think it's easier to understand and, more importantly, add/remove sanitizers should the need arise.

Reply:
This looks much better to me. I see you're using it [here](https://github.com/systemd/systemd/pull/16150/files#diff-4a0e55c02d866fe39ac44f409c686cf1) so I imagine this "just works". Good idea, let's change the example.